docs/stdlib/datetime.rst

```diff
@@ -905,6 +903,77 @@ EdgeDB stores and outputs timezone-aware values in UTC.
         ...   <duration>'30 hours', 'totalseconds');
         {108000}
 
+    .. rubric:: Gotchas
+
+    This function will provide you with a calculated total for the unit passed
+    as ``el``, but only within the given "size class" of the unit. The size
+    classes are as follows:
+
+    - ``'month'`` and larger
+    - ``'day'``
+    - ``'hour'`` and smaller
+
+    For example, if you specify ``'day'`` as your ``el`` argument, the function
+    will return only the number of days expressed as ``N days`` in your
+    duration. It will not add another day to the returned count for every 24
+    hours (defined as ``24 hours``) in the duration, nor will it consider the
+    months' constituent day counts in the returned value.
+
+    Specifying ``'decade'`` for ``el`` will total up all decades represented in
+    units ``'month'`` and larger, but it will not add a decade's worth of days
+    to the returned value as an additional decade. On the other end of the unit
+    spectrum, passing ``'hour'`` for ``el`` will not add a day's worth of hours
+    for each day in your duration.
+
+    In this example, the duration represents more than a day's time, but since
+    ``'day'`` and ``'hour'`` are in different size classes, the extra day
+    stemming from the duration's hours is not added.
+
+    .. code-block:: edgeql-repl
+
+        db> select duration_get(
+        ...   <cal::relative_duration>'1 day 36 hours', 'day');
+        {1}
+
+    In this counter example, both the decades and months are pooled together
+    since they are in the same size class. The return value is 5: the 2
+    ``'decades'`` and the 3 decades in ``'400 months'``.
+
+    .. code-block:: edgeql-repl
+
+        db> select duration_get(
+        ...   <cal::relative_duration>'2 decades 400 months', 'decade');
+        {5}
+
+    If a unit from a smaller size class would contribute to your desired unit's
+    total, it is not added.
+
+    .. code-block:: edgeql-repl
+
+        db> select duration_get(
+        ...   <cal::relative_duration>'1 year 400 days', 'year');
+        {1}
+
+    When you request a unit in the smallest size class, it will be pooled with
+    other durations in the same size class.
+
+    .. code-block:: edgeql-repl
+
+        db> select duration_get(
+        ...   <cal::relative_duration>''20 hours 3600 seconds', 'hour');
```

Comment:
This apparently has a test failing over ``seconds`` being used. This looks completely fine though, is there something wrong with the test invalidating?

Reply:
I'm guessing that was a result of my errant quotes that you found? (Thanks for catching that, btw.) It looks like all the tests are passing now, but let me know if you still see an issue.